packages/devtools_app/test/test_infra/test_data/network.dart

```diff
@@ -88,7 +88,7 @@ final Map<String, dynamic> testSocket1Json = {
 
 final testSocket2 = WebSocket(SocketStatistic.parse(testSocket2Json)!, 0);
 final Map<String, dynamic> testSocket2Json = {
-  'id': 1,
+  'id': '11111',
```

Comment:
To ensure the websocket Id's don't override httprequest ids, I've given then different ids :)